.github/workflows/push.yml

```diff
@@ -164,4 +164,28 @@ jobs:
         user_email: '${{env.email}}'
         user_name: '${{github.actor}}'
         commit_message: 'Talawa API docs updated'
-      
+        
+# You can find the deployment instructions in the scripts/cloud-api-demo/README.md file
+  Deploy-Workflow:
+    name: Deploying Application to Cloud VPS
+    runs-on: ubuntu-latest
```

Comment:
Add this so that it only runs after tests pass.

 `needs: Push-Workflow`

Reply:
Done